docs/pages/access-controls/sso/github-sso.mdx

```diff
@@ -121,6 +121,21 @@ $ tctl create github.yaml
   able to determine team memberships for these organizations.
 </Admonition>
 
+After authentication Teleport will create a temporary GitHub type user within Teleport for the duration of the 
```

Comment:
I don't love this sentence.

- "Teleport will create a user within Teleport" is a little confusing
- "GitHub type user" is not a term I'd expect the reader to understand

Reply:
Removed and simplified language.